tests/test_viewmodel.py

```diff
@@ -0,0 +1,68 @@
+"""Unit tests for viewmodel.py"""
+
+from email.errors import InvalidBase64PaddingDefect
+import pytest
+from todo_app import viewmodel
+from todo_app.data.trello_items import Item
+
+@pytest.fixture
+def view_model():
+    items = []
+    items.append(Item(1, 'To Do', 'item1'))
+    items.append(Item(2, 'To Do', 'item2'))
+    items.append(Item(3, 'Doing', 'item3'))
+    items.append(Item(4, 'Doing', 'item4'))
+    items.append(Item(5, 'Doing', 'item5'))
+    items.append(Item(6, 'Done', 'item6'))
+    items.append(Item(7, 'Done', 'item7'))
+    items.append(Item(8, 'Done', 'item8'))
+    items.append(Item(9, 'Done', 'item9'))
+    return viewmodel.ViewModel(items)
+
+def test_view_model_todo_items(view_model: viewmodel.ViewModel):
+    # arrange / act
```

Comment:
I like that you're explicitly calling these stages out! Arrange can often be a little odd when it's really all done in the fixture like here, but I think this is clear as is